tests/unit/GatewayOrderTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+class GatewayOrderTest extends TestCase
+{
+    protected $instaobj;
+
+    public function setUp()
+    {
+        $this->instaobj = Instamojo\Instamojo::init('app',[
+            "client_id" => $_ENV["CLIENT_ID"],
+            "client_secret" => $_ENV["CLIENT_SECRET"]
+        ],true);
+    }
+
+    public function test_create_a_gateway_order()
+    {
+        $transaction_id = "TEST_".time();
+        
+        $gateway_order = $this->instaobj->createGatewayOrder([
+            "name" => "XYZ",
+            "email" => "xyz@squareboat.com",
```

Comment:
remove squareboat references from here and all other code files. Make this example.com